app/services/repository_service.py

```
@@ -0,0 +1,46 @@
+
+from app.repositories.git_label_repository import TortoiseGitLabelStore
+from app.repositories.repo_repository import TortoiseRepoStore
+from app.schemas.basic import PaginationParams
+from app.schemas.repo import RepoResponse
+from app.utils.auth import UserClaims
+
+
+class RepoQueryService:
+    def __init__(
+        self,
+        repo_store: TortoiseRepoStore = Depends(),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)